### PR TITLE
update for tms connected and update url for image

### PR DESCRIPTION
**Pull request by Rahul139-stack**
Please review and approve it.
image

### CODE REVIEW

File: application/backend/src/main/resources/apps/typerefinery/components/widgets/map/clientlibs/functions.js [hunk 1]
```diff
@@ -60,13 +67,33 @@ window.Typerefinery.Page.Tms = Typerefinery.Page.Tms || {};
...
+                if (layer instanceof leafletMap.Marker){
+                    map.removeLayer(layer)
+                }                
+        })
+        if(data.mapLat && data.mapLng && data.zoomLevel){
+            map.setView([data.mapLat, data.mapLng ], data.zoomLevel);
+        }
+        const markerDetails=data.markers
+        if(markerDetails.length>0){
+            ns.addMarker(data.markers, map)
```
Comment (Harikarthyk) [inline application/backend/src/main/resources/apps/typerefinery/components/widgets/map/clientlibs/functions.js:84]: You are taking up data.markers and assigning it to markerDetails,
So why again using data.markers ?

**Review by Harikarthyk — CHANGES_REQUESTED** [review]
I could not see the dialog box updates in the map component

File: application/frontend/src/static/clientlibs/vendor/map/leafletmap/leaflet.css [hunk 2]
```diff
@@ -399,7 +399,7 @@ svg.leaflet-image-layer.leaflet-interactive path {
 
 /* Default icon URLs */
 .leaflet-default-icon-path { /* used only in path-guessing heuristic, see L.Icon.Default */
-	background-image: url("http://localhost:8080/apps/typerefinery/clientlibs/vendor/map/leafletmap/images/marker-icon.png");
+	background-image: url("http://localhost:8113/apps/typerefinery/clientlibs/vendor/map/leafletmap/images/marker-icon.png");
```
[the remark this answers is not in the document]
Comment (Rahul139-stack) [reply application/frontend/src/static/clientlibs/vendor/map/leafletmap/leaflet.css:402]: yes we can, updated in next commit

File: application/backend/src/main/resources/apps/typerefinery/components/widgets/map/clientlibs/functions.js (hunk 1 above)
Comment (Rahul139-stack) [reply application/backend/src/main/resources/apps/typerefinery/components/widgets/map/clientlibs/functions.js:84]: Replaced

**Comment by Rahul139-stack** [comment]
> I could not see the dialog box updates in the map component

Done

**Review by Harikarthyk — CHANGES_REQUESTED** [review]
image

Have same name and keys in the dialog box.

**Comment by Rahul139-stack** [comment]
> image
> 
> Have same name and keys in the dialog box.

Renamed as expected

File: application/backend/src/main/resources/apps/typerefinery/components/widgets/map/dialog/.content.json [hunk 3]
```diff
@@ -5,13 +5,13 @@
     "configTab": {
```
Comment (Harikarthyk) [inline application/backend/src/main/resources/apps/typerefinery/components/widgets/map/dialog/.content.json:5]: why the name and key is different here ?

**Review by Harikarthyk — CHANGES_REQUESTED** [review]
image

You can make the key as generalTab

File: application/backend/src/main/resources/apps/typerefinery/components/widgets/map/dialog/.content.json (hunk 3 above)
Comment (Rahul139-stack) [reply application/backend/src/main/resources/apps/typerefinery/components/widgets/map/dialog/.content.json:5]: Done

**Review by Harikarthyk — APPROVED** [review]

**Pull request merged**